Form-Controls/index.html

```diff
@@ -14,14 +14,52 @@ <h1>Product Pick</h1>
   </header>
   <main>
   <form>
-    <!-- write your html here-->
-    <!-- try writing out the requirements first-->
+    <div>
+      <label for="name">Name</label>
+      <input type="text" id="name" name="name" required pattern="[A-Za-z\s]{2,}" title="Please enter at least 2 letters.">
```

Comment:
Hi @Yulia-I19 ,
Just a nitpick, how about replacing the name with username for readability. Also, why use _title_ attribute in the input field? I think you meant to use **placeholder**